src/SonarPartTwo.php

```diff
@@ -6,10 +6,20 @@ class SonarPartTwo
 {
     function SweepPartTwo(string $puzzleInput): int
     {
-        return 6;
+        $puzzleInput = explode("\n", $puzzleInput);

```

Comment:
puzzleInput?

Reply:
puzzleInput is the input data from the advent of code website. Did you want the name changing?

Reply:
doesn't have anything to do with a submarine

Reply:
Agreed. Name changed to subData.